config/envs/unit_test.py

```diff
@@ -15,3 +17,16 @@ class UnitTestConfig(DefaultConfig):
     EXAMPLE_INGEST_WRONG_FORMAT = DefaultConfig.FLASK_ROOT + "/tests/resources/wrong_format_test_file.txt"
     EXAMPLE_INGEST_DATA_PATH = DefaultConfig.FLASK_ROOT + "/tests/resources/Pre_ingest_EXAMPLE_data.xlsx"
     DATA_STORE_API_HOST = "http://data-store"
+    ADDITIONAL_EMAIL_LOOKUPS = os.environ.get(
+        "PRIVATE CONTRACTOR LOOKUPS",
+        {
+            "version1.com": (
+                ("Sunderland City Council", "Worcester City Council"),
+                ("Sunderland City Centre", "Blackfriars - Northern City Centre", "Worcester"),
```

Comment:
I think it might be best if we can submit for any LA... this may be a bit restrictive for testing etc. What do you think?

Reply:
The thinking here is that this replicates actual user behaviour more accurately, and that we can update the places we want to test submission for as we go. The alternatives are either passing a massive tuple of every single place name, or a bodge in which we write bespoke code to enable fsd folk to submit for anywhere (e.g. setting version1 emails to submit for (("All",), ("All",)), and then writing if place_names == "All", but both feel a bit hacky.

Happy to hear other suggestions?

Reply:
Yeah I get you, makes sense. If we can test both HS and TD with these place names then I guess that's probably fine. If not we can revisit.

Reply:
Also we can still submit for anywhere via the API, this just restricts submissions for the Submit tool